src/chunked.rs

```diff
@@ -10,19 +10,27 @@ use gfx_hal::memory::Requirements;
 use {alignment_shift, MemoryAllocator, MemoryError, MemorySubAllocator};
 use block::{Block, RawBlock};
 
+/// Chunks is super-allocator block
+/// That is devided into smaller blocks
```

Comment:
as this starts with capital letter, we need to end the previous sentence with a period.